engine/flink/components/base/src/test/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/DailyWindowDetectorSpec.scala

```diff
@@ -0,0 +1,30 @@
+package pl.touk.nussknacker.engine.flink.util.transformer.aggregate
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.duration.Duration
+
+class DailyWindowDetectorSpec extends AnyFunSuite with Matchers {
+
+  test("window 10m - is not daily window") {
```

Comment:
is not a daily window? And other below too

Reply:
it's not - its 10 minutes

Reply:
I meant there is "a" missing :woman_dancing: